### PR TITLE
Link update

### PR DESCRIPTION
**Pull request by Nadine2016**

### CODE REVIEW

File: content/portworx-install-with-kubernetes/cloud/ibm/_index.md [hunk 1]
```diff
@@ -1,82 +1,80 @@
 ---
-title: IKS
+title: IBM Cloud Kubernetes Service
 logo: /logos/ibm.png
-linkTitle: IKS
+linkTitle: IBM Cloud Kubernetes Service
```
Comment (harsh-px) [inline content/portworx-install-with-kubernetes/cloud/ibm/_index.md]: @Nadine2016 can you have a shorter title than IBM Cloud Kubernetes Service. How about IBM IKS or IBM Cloud? We are already in the Kubernetes section so the "Kubernetes Service" part can be skipped.

This doens't look great on the sidebar and also distorts the formatting of the tiles. Screenshot below.

screen shot 2019-02-21 at 12 45 31 pm

**Comment by Nadine2016** [comment]
@harsh-px I changed to IBM Cloud. Unfortunately IKS is not approved to be used by our legal department

**Comment by harsh-px** [comment]
Thanks for updating the linkTitle. It would also like the title to be the same. The current long title distorts the tile on https://docs.portworx.com/portworx-install-with-kubernetes/cloud/. Screenshot below.

screen shot 2019-02-21 at 2 20 17 pm

**Comment by Nadine2016** [comment]
@harsh-px I updated the title to IBM Cloud as well.

**Pull request merged**